src/ucp/core/ucp_request.h

```diff
@@ -52,6 +53,17 @@ enum {
         } \
     }
 
+/*
+ * - Mark the request as completed
+ * - Otherwise - the request might be released - if so, return it to mpool.
+ */
+#define ucp_request_rma_complete(_req) \
```

Comment:
i believe you can pass 'void' as the callback parameter to this macro to avoid calling a callback.